### PR TITLE
Components

### PR DESCRIPTION
**Pull request by brytebee**
## Components structure 🏛️🏛️
In this PR, I have accomplished the following:
- created the repo following standard Git flow
- added the structure of the components
- added linter files and GitHub action
- added routes for the app

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @brytebee 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/BookCard.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import './BookCard.css';
+
+export default function BookCard() {
+  const books = [
+    {
+      author: 'Robert Kiyosaki',
+      title: 'Rich Dad, Poor Dad',
+      category: 'Finance',
```
Comment (mikenath223) [inline src/redux/books/BookCard.js]: - [ ] Please move **all** the components under the src/redux folder to a components folder.

File: src/redux/books/BookCard.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      </div>
+    </div>
+  );
+}
+
+// BookCard.propsTypes = {
+//   category: PropTypes.string,
+//   title: PropTypes.string,
+//   author: PropTypes.string,
+// };
```
Comment (mikenath223) [inline src/redux/books/BookCard.js]: - [x] Please remove this commented code block

Comment (brytebee) [reply src/redux/books/BookCard.js]: Removed commented code

File: src/redux/books/BookCard.js (hunk 1 above)
Comment (brytebee) [reply src/redux/books/BookCard.js]: Updated the files structure.

**Review by dasileker — APPROVED** [review]
## **Hi @brytebee ,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**